src/components/views/dialogs/DevtoolsDialog.js

```diff
@@ -412,13 +412,13 @@ class RoomStateExplorer extends React.PureComponent {
         if (this.state.eventType === null) {
             list = <FilteredList query={this.state.queryEventType} onChange={this.onQueryEventType}>
                 {
-                    Object.keys(this.roomStateEvents).map((evType) => {
-                        const stateGroup = this.roomStateEvents[evType];
-                        const stateKeys = Object.keys(stateGroup);
+                    Array.from(this.roomStateEvents.keys()).map((evType) => {
+                        const stateGroup = this.roomStateEvents.get(evType);
```

Comment:
I would argue this should use `entries`

Reply:
fair, was going for minimal touch points on the first pass.